woocommerce/woocommerce
# Added API testing package

### Changes proposed in this Pull Request:

This PR introduces a new API testing package for testing the WooCommerce API.

### How to test the changes in this Pull Request:

1. In the `woocommerce/tests/e2e/api-core-tests` directory, run `npm install` to pull in the required packages.
2. Set up a test site to run the API tests against.
3. Follow the instructions in the README file to setup and run the tests.
4. Verify the tests run successfully.
5. Optionally, create a Postman collection and import the generated `collection.json` file into Postman and verify the endpoints are imported successfully.
